spec/integration/inventory_spec.rb

```diff
@@ -279,8 +319,11 @@ module Hawkular::Inventory::RSpec
       resource_path = CanonicalPath.new(feed_id: feed_id, resource_ids: [hawk_escape_id('Local~~')])
       config = @client.get_config_data_for_resource(resource_path)
 
+      # product name is missing
       expect(config['value']['Server State']).to eq('running')
-      expect(config['value']['Product Name']).to eq('Hawkular')
```

Comment:
This and similar is a bit unfortunate, as the change is not specific to inventory, but rather to the old recording against a version that identified as Hawkular vs what we have currently.

Similar for 'ExampleDS' vs 'Datasource[ExampleDS]'  If we would re-record against a current server with the old api, we would see ''Datasource[ExampleDS]'  in both cases.